### PR TITLE
Abstract out AddInstrumentParserTest

### LINKED ISSUE #42: Abstract all common test classes' methods into another class file
Seems like we are repeating similar pieces of code in the test classes for different parsers. We may want to consider having all of them in a separate file.

In addition standardise and use the SEPARATOR_SPECIFIER for the add instrument testing

### PR DESCRIPTION
**Pull request by KVignesh122**
Code Quality improvement: Abstract all common test classes' methods into another class file.

Fix #42

### CODE REVIEW

File: src/test/java/seedu/mtracker/console/AddCryptoParserTest.java [hunk 1]
```diff
@@ -69,29 +64,17 @@
...
-            assertEquals(testCryptoParser.getParameters().get(i), expectedParameters[i]);
-        }
-    }
-
     void testCryptoParameters(String input, String[] expectedParameters) {
         simulateConsoleInput(input);
         AddCryptoParser testCryptoParser = new AddCryptoParser();
         testCryptoParser.initParameters();
         testCryptoParser.getInstrumentParameters();
         checkParameters(testCryptoParser, expectedParameters);
```
Comment (theodorekwok) [inline src/test/java/seedu/mtracker/console/AddCryptoParserTest.java:72]: is there a reason why this is not refactored like the rest of the other test classes?

**Review by theodorekwok — COMMENTED** [review]
Nice work just one comment.

File: src/test/java/seedu/mtracker/console/AddCryptoParserTest.java (hunk 1 above)
Comment (KVignesh122) [reply src/test/java/seedu/mtracker/console/AddCryptoParserTest.java:72]: oops, I forgot.

**Review by theodorekwok — APPROVED** [review]
LGTM!

**Pull request merged**